jsapp/scss/components/_kobo.modal.scss

```diff
@@ -484,6 +484,281 @@
   }
 }
 
+.form-modal--translation-settings {
+  .form-view__cell--translation-note {
+    padding-bottom: 20px;
+  }
+  .form-view__cell--translation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-top: 1px solid $divider-color;
+    padding: 6px 0px;
+  }
+
+  .form-view__cell--translation-actions {
+    min-width: 150px;
+    text-align: right;
+  }
+
+  .form-view__cell--add-language {
+    padding-top: 20px;
+    text-align: right;
+    border-top: 1px solid $divider-color;
+  }
+
+  .form-view__cell--add-language-form,
+  .form-view__cell--update-language-form {
+    background: #F8F8F8;
+    padding: 15px;
+    position: relative;
+
+    .form-view__link--close {
+      position: absolute;
+      right: 0px;
+      top: 6px;
+
+      i {
+        font-size: 24px;
+      }
+    }
+  }
+
+  .form-view__cell--add-language-form {
+    margin-top: 20px;
+  }
+
+  .form-view__cell--add-language-fields {
+    display: flex;
+    justify-content: space-between;
+
+    .form-view__cell {
+      min-width: 100px;
+
+      &--lang-code {
```

Comment:
It's just my preference, but when I try to find `form-view__cell--lang-code` when debugging in Dev Tools, this notation makes it so much harder